components/Sidebar/index.jsx

```diff
@@ -0,0 +1,46 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+
+import useUser from 'hooks/useUser';
+import { clearToken } from 'helpers/authentication';
+
+export const sidebarTestIds = {
+  base: 'sidebar',
+  userName: 'sidebar__user-name',
+  button: 'sidebar__logout-button',
+};
+
+const Sidebar = ({ name }) => {
+  const { mutateUser } = useUser({
+    redirectTo: '/login',
+  });
+
+  const handleOnClick = async event => {
+    event.preventDefault();
+    clearToken();
+    mutateUser();
+  };
+
+  return (
+    <div
+      className="fixed bg-sidebar backdrop-blur-4xl right-0 w-[264px] pl-8 pt-8 pr-8 h-full"
+      data-test-id={sidebarTestIds.base}
+    >
+      <div className="text-white text-base-xxl font-extrabold mb-[22px]" data-test-id={sidebarTestIds.userName}>
```

Comment:
Should we make this center align with the avatar, maybe adjust the height and alignment of this? 💭 

The current style | The design
--- | ---
![Screen Shot 2565-04-11 at 15 15 56](https://user-images.githubusercontent.com/1772999/162694321-a97702c3-6dbd-4bc6-87f3-d7afee67c702.png) | ![Screen Shot 2565-04-11 at 15 15 30](https://user-images.githubusercontent.com/1772999/162694336-38e44373-c7db-4a98-a1ef-07c9dc5a0486.png)

Reply:
done in 2125544666199978509632db13f61e3cd5903fa6